Makefile

```diff
@@ -323,10 +322,9 @@ lint-fix:
 ##                                 Protobuf                                  ##
 ###############################################################################
 
-DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace bufbuild/buf:1.8.0
-
-protoVer=v0.7
-protoImageName=tendermintdev/sdk-proto-gen:$(protoVer)
+protoVer=0.13.0
```

Comment:
I think this is not compatible with cosmos SDK v0.46

Reply:
Ohh, we will wait for util v0.47 integration to our repo